### DISCUSSION TITLE
Can we set a fixed client subnet for ECS?

### DISCUSSION THREAD

**Discussion by timkgh** [AdguardTeam/AdGuardHome#3649] [Q&A] [answered]
Your `dnsproxy` allows `--edns-addr=PUBLIC_IP`. Does the same feature exist in AGH or can it be added please?
I'd like to set it to the IP of the 1st hop from my WAN IP, that of an ISP router to still get geo locality but not expose my actual IP.

And also it should support passing along any ECS that comes from the client, if AGH (and dnsproxy) serve clients from different locations that pass their own ECS. I added this issue: https://github.com/AdguardTeam/dnsproxy/issues/166

**Comment by ameshkov** [comment, marked as answer]
Looks like this? https://github.com/AdguardTeam/AdGuardHome/issues/2514

**Comment by timkgh** [comment]
Any ideas in terms of priority? It's a feature from `dnsmasq` that I miss.

**Reply by ameshkov** [reply]
Planned on v0.108 release.
